src/App.css

```diff
@@ -14,5 +14,5 @@
 }
 
 .border-danger {
-  border: 1px solid var(--danger-color);
+  border: 1px solid var(--danger-color) !important;
```

Comment:
Avoid using **`!important`** flag in CSS rules. Try to write more specific CSS rule.

Reply:
I can still see **`!important`**.

Reply:
Sorry, I missed that.. I fixed it now.